locations/storefinders/rio_seo.py

```diff
@@ -38,7 +38,14 @@ def parse_autocomplete(self, response: Response, **kwargs: Any) -> Any:
 
     def parse(self, response, **kwargs):
         map_list = response.json()["maplist"]
-        data = json.loads("[{}]".format(Selector(text=map_list).xpath("//div/text()").get()[:-1]))
+        try:
+            data = json.loads("[{}]".format(Selector(text=map_list).xpath("//div/text()").get()[:-1]))
+        except json.decoder.JSONDecodeError:
+            self.logger.warning("Could not parse response - check API output")
```

Comment:
This reminded me that the project should document when to use WARNING and ERROR, which at the moment I think is fairly random in ATP.

In this case, I suggest perhaps `self.logger.error` is more appropriate than `self.logger.warning` because the spider will fail to return any results if this branch is encountered.

Reply:
I chose warning because it will fail.. for that URL

Reply:
> I chose warning because it will fail.. for that URL

Ah sorry I forgot this storefinder supports more than one start URL :)